assets/src/util/roles.js

```diff
@@ -0,0 +1,24 @@
+/**
+ *  Role utility class
+ * */
+
+/**
+  * Array of string values that are functionally equivalent in representing 'instructor' role
+  */
+const instructorRoles = [
+  'TeacherEnrollment', // student_dashboard.user enrollment_type
+  'Instructor' // LTI role
+]
+
+/**
+ * Return true if the provided criteria represent the role of an a
```

Comment:
Is this comment correct?

Reply:
Is any comment, really?

But in this case no.